lib/commands/run.js

```diff
@@ -332,7 +332,8 @@ function readPayload(script, callback) {
     script.config.payload,
     function readPayloadFile(payloadSpec, next) {
       let data = fs.readFileSync(payloadSpec.path, 'utf-8');
-      csv(data, function(err, parsedData) {
+      const csvOpts = Object.assign({}, payloadSpec.options);
```

Comment:
it might be more idiomatic to write this as:
`const csvOpts = payloadSpec.options || {};`